container_files/tutorials/Virtual manipulation of datasets.ipynb

```diff
@@ -0,0 +1,924 @@
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>neural</th>\n",
+       "      <td>6</td>\n",
+       "      <td>6</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>training</th>\n",
+       "      <td>5</td>\n",
+       "      <td>5</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>model</th>\n",
+       "      <td>5</td>\n",
+       "      <td>5</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>network</th>\n",
+       "      <td>5</td>\n",
+       "      <td>5</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>used</th>\n",
+       "      <td>5</td>\n",
+       "      <td>5</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>machines</th>\n",
+       "      <td>5</td>\n",
+       "      <td>5</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>artificial</th>\n",
+       "      <td>4</td>\n",
+       "      <td>4</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "            top_words_transp_dataset  top_words_transp_function\n",
+       "_rowName                                                       \n",
+       "learning                           7                          7\n",
+       "with                               7                          7\n",
+       "machine                            6                          6\n",
+       "neural                             6                          6\n",
+       "training                           5                          5\n",
+       "model                              5                          5\n",
+       "network                            5                          5\n",
+       "used                               5                          5\n",
+       "machines                           5                          5\n",
+       "artificial                         4                          4"
+      ]
+     },
+     "execution_count": 16,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "mldb.query(\n",
+    "    \"\"\"\n",
+    "        SELECT \n",
+    "            horizontal_count({a.*}) AS top_words_transp_dataset,\n",
+    "            horizontal_count({b.*}) AS top_words_transp_function\n",
+    "        NAMED a.rowName()\n",
+    "        FROM transposed_concepts AS a\n",
+    "        JOIN transpose(ml_concepts) AS b\n",
+    "            ON a.rowName() = b.rowName()\n",
+    "            ORDER BY top_words_transp_dataset DESC\n",
+    "            LIMIT 10\n",
+    "    \"\"\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "While the methods yield similar results, the transposed dataset allows the user to keep the output datable in memory. This may be useful if you want to use the dataset in multiple steps of your data analysis."
```

Comment:
datable -> dataset

et ajouter: In cases where you don't need to use the transpose more than once, you can simply use the inline syntax.